crest/ProjectSettings/ProjectSettings.asset

```diff
@@ -620,7 +620,7 @@ PlayerSettings:
   webGLThreadsSupport: 0
   webGLDecompressionFallback: 0
   scriptingDefineSymbols:
-    1: CROSS_PLATFORM_INPUT;UNITY_POST_PROCESSING_STACK_V2;CREST_OCEAN
+    1: CROSS_PLATFORM_INPUT;UNITY_POST_PROCESSING_STACK_V2;CREST_OCEAN;CREST_DEBUG
```

Comment:
just double checking this is intentional? not objecting just checking

Reply:
It is. For convenience since FO testing is wrapped with this.